pkg/types/defaults/installconfig.go

```diff
@@ -107,5 +108,8 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 		}
 	case c.Platform.None != nil:
 		nonedefaults.SetPlatformDefaults(c.Platform.None)
+	case c.Platform.Nutanix != nil:
+		nutanixdefaults.SetPlatformDefaults(c.Platform.Nutanix, c)
```

Comment:
The install config is not used. Do not pass it in.
```suggestion
		nutanixdefaults.SetPlatformDefaults(c.Platform.Nutanix)
```

Reply:
ok